src/store/gameActions.js

```diff
@@ -0,0 +1,74 @@
+export const types = {
```

Comment:
This file seems to have been recreated? It was removed in the refactor, everything should go with components now

Reply:
Oh yeah, it's not used but the merging still added it, I'll delete it now